plans/utils/inventory_yaml.pp

```diff
@@ -0,0 +1,16 @@
+# @summary Destroy a pecdm provisioned PE cluster
```

Comment:
I know picking at comments is just fussy but its still saying destroy not deply

Reply:
I've fixed them all in local copy that I've not yet pushed.